config/config.js

```diff
@@ -28,5 +28,7 @@ module.exports = Object.freeze({
 
   defaultGasPrice: parseInt(process.env.DEFAULT_GAS_PRICE, 10) || 5, // in ambits
 
-  gitCommit: process.env.GIT_COMMIT
+  gitCommit: process.env.GIT_COMMIT,
+
+  logsPath: process.env.LOGS_PATH || 'server-logs.log'
```

Comment:
It would be good if this defaulted to stdout because logging in Docker means that any process in the container emits logs to stdout.